src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write your code here
+
+const total = document.querySelector('.total-population');
+const average = document.querySelector('.average-population');
+
+const population = document.getElementsByClassName('population');
```

Comment:
Consider using `document.querySelectorAll` instead of `getElementsByClassName` for a more modern approach that returns a static NodeList, which can be directly iterated with forEach, for...of, or converted to an array if needed.